### PR TITLE
feat: Add codeFrame to record data attempt errors

### PR DESCRIPTION
**Pull request by chrisbreiding**
Internal only - no need for inclusion in changelog.

Attaches the codeFrame property to attempt errors in data sent to the dashboard.

**DO NOT MERGE** until dashboard json schema has been updated to accept this new property.

### PR Tasks

- [x] Have tests been added/updated?
- N/A Has the original issue been tagged with a release in ZenHub? 
- N/A Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- N/A Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- N/A Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: cli/types/cypress-npm-api.d.ts [hunk 1]
```diff
@@ -8,10 +8,22 @@
...
+    absoluteFile: string
+    frame: string
+    language: string
+  }
+
   interface TestError {
     name: string
     message: string
     stack: string
+    codeFrame?: CodeFrame
```
Comment (flotwig) [inline cli/types/cypress-npm-api.d.ts:26]: so should this be in the changelog?

Comment (flotwig) [reply cli/types/cypress-npm-api.d.ts:26]: since it affects public types

Comment (chrisbreiding) [reply cli/types/cypress-npm-api.d.ts]: Yeah, I guess the type changes weren't necessary after all, so no need to add anything to the changelog.

**Review by flotwig — APPROVED** [review]

**Comment by chrisbreiding** [comment]
@estrada9166, has the dashboard schema been updated to accept the `codeFrame` property?

**Comment by estrada9166** [comment]
@chrisbreiding it is ready now, and actually we have already one case that recorded those values for this branch.
[Dashboard](https://dashboard.cypress.io/projects/ypt4pf/runs/12644/test-results/0c8ca20e-2dbd-4b40-8d04-20e571c69c8e)

**Comment by chrisbreiding** [comment]
@estrada9166 Great! This will get released with the next version then, which should be on Oct 13.

**Pull request merged**